### PR TITLE
Allow redis config template to be sourced from another cookbook

### PR DESCRIPTION
**Pull request by veurits**
I needed to change the template, and easiest way to do this was to change the cookbook/name. It wasn't configurable.

### CODE REVIEW

**Comment by martinb3** [comment]
If it was [added in the resource](https://github.com/brianbianco/redisio/blob/master/resources/configure.rb#L33), we could support different templates from different cookbooks per instance.

File: providers/configure.rb [hunk 1]
```diff
@@ -170,8 +170,8 @@ def configure
       
       #Lay down the configuration files for the current instance
       template "#{current['configdir']}/#{server_name}.conf" do
-        source 'redis.conf.erb'
-        cookbook 'redisio'
+	source node['redisio']['template_source']
```
Comment (shortdudey123) [inline providers/configure.rb]: spaces not tabs

File: attributes/default.rb [hunk 2]
```diff
@@ -31,6 +31,10 @@
   homedir = '/redis'
 end
 
+# Overwite template
+default['redisio']['template_cookbook'] = "redisio"
```
Comment (shortdudey123) [inline attributes/default.rb]: single quotes

**Comment by shortdudey123** [comment]
Carried by https://github.com/brianbianco/redisio/pull/341

**Pull request closed without merging**